AlphaWallet/Core/Initializers/MigrationInitializer.swift

```diff
@@ -92,8 +92,8 @@ extension MigrationInitializer {
 
     //We use the existence of realm databases as a heuristic to determine if there are wallets (including watched ones)
     static var hasRealmDatabasesForWallet: Bool {
-        let documentsDirectory = URL(fileURLWithPath: NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)[0])
```

Comment:
How do we figure out that `RealmConfiguration.defaultRealmFolderUrl` returns the same value as the old one (`URL(fileURLWithPath: NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)[0])`) ?

Reply:
Especially when not running tests.

Reply:
Ok